packages/react-components/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -77,6 +77,10 @@ export interface ActionMenuProps {
    * https://floating-ui.com/docs/usefloating#strategy
    */
   floatingStrategy?: Strategy;
+  /**
+   * Test id passed to the wrapper element
+   */
+  ['data-testid']?: string;
```

Comment:
@VadymBezpalko Have you considered an extendable Props interface for all components, so some core fields would be part of every single component? `['data-testid']` could be a part of them. 

Such approach would prevent you from necessity to add them from time to time when needed.

Reply:
Great idea 👍